frontend/src/shared/components/ProgressBar.tsx

```diff
@@ -0,0 +1,29 @@
+interface ProgressBarProps {
+  maxStep: number;
+  currentStep: number;
+}
+
+interface ProgressBarItemProps {
+  filled: boolean;
+}
+
+const ProgressBar = ({ maxStep, currentStep }: ProgressBarProps) => {
+  const steps = Array.from({ length: maxStep }, (_, index) => index + 1);
+
+  return (
+    <div className='flex w-full items-center gap-[1rem]'>
```

Comment:
```suggestion
    <div className='flex-center w-full gap-[1rem]'>
```

요렇게 만들어진 tailwind utilities 사용할 수 있을 것 같아요~

Reply:
네네 `justify-content:center`속성은 필요가 없었어서 그렇게 했지만, 사용하는게 더욱 간결해질 것 같습니다!
gap은 figma에서 1rem보다 작아야한다는 점 확인하여, 말씀하신 수정사항 반영하였습니다~